si-id-epu50/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include si_id_conv/access_rules.as
```

Comment:
typo here si_id_epu50

Reply:
thanks, @anacso17 . the issue is deprecated in new commits